CHANGELOG.md

```diff
@@ -87,6 +87,7 @@ administrators should keep in mind updating the help texts for each step.
 
 **Fixed**:
 
+- **decidim-core**: Fixes the linked_resources_for method to only show the resources that has the component published. [\#6](https://github.com/CodiTramuntana/decidim/pull/6)
```

Comment:
Wrong PR data

Reply:
You are right! It was the data of the first PR.. Can you check now? Keep in mind, that you have to do a PR to master. We did this PR because we needed it urgently in an Decidim installation 😃